sqlite-db/src/rollover/insert.rs

```diff
@@ -19,61 +17,38 @@ use sqlx::Transaction;
 pub async fn insert(
```

Comment:
🙃Since this is not just `insert`ing data but also deleting, should we rename it `replace_rollover`. Not sure it makes it better, but since I noticed I thought I mention it.

Reply:
I've pushed a few commits in this direction.